src/Speck.js

```diff
@@ -134,13 +134,32 @@ class Speck {
     return JSON.parse(JSON.stringify(rawData));
   }
 
+  toJSONContext(context) {
```

Comment:
Even though I liked the approach of using `getContext()` on it's on, what it will return is a real issue. If you really liked this way, maybe `setContext` on an entity would be more appropriate, since it could be chained by returning itself, like:
`user.setContext('login').validate()` or `user.setContext('atronaut').toJSON()`

The problem would be **unseting** it. I don't think that make much sense either... or maybe every entity should always be contextualized?

Anyway, I think I prefer the `user.toJSON({ context })` usage, since it could be extendable for future tweaks (or maybe thats a bad thing? 🤔). 
`.validateContext()` should be normalized accordingly